rc/lsp.kak

```diff
@@ -1175,22 +1175,6 @@ buf_line_count = ${kak_buf_line_count}
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
-define-command -hidden lsp-experimental-inlay-hints -docstring "lsp-experimental-inlay-hints: Request inlay hints with experimental prefix" %{
-  lsp-did-change-and-then lsp-experimental-inlay-hints-request
-}
-
-define-command -hidden lsp-experimental-inlay-hints-request %{
```

Comment:
Very cool. Can you also remove `lsp-experimental-inlay-hints-enable` and `lsp-experimental-inlay-hints-disable`?